apps/system/flash_test/Kconfig

```diff
@@ -3,9 +3,9 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-config SYSTEM_FLASH_ERASEALL
-	bool "FLASH eraseall tool"
+config SYSTEM_FLASH_TEST
```

Comment:
There is no test stub. It only has read operation. Let's rename it.
 `SYSTEM_FLASH_READ`